### DISCUSSION TITLE
Disk failure - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#55] [General]
# Disk failure - Alan Pope's blog

As regular readers of my blog may recall, I recently gave my brother a PC running Ubuntu Linux. He uses it for surfing the web, getting email and chatting online. He also has an HP PSC 1610 printer/scanner/copier which works under Linux pretty well. He has a Dlink WAP/Modem/Router providing his ADSL connection via wired ethernet.
I had a phone call from him a week or so ago telling me the PC was broken.

https://popey.com/blog/2006/02/disk-failure/

**Comment by Morgenkaff** [comment]
A custom live ISO, with your public SSH key, SSH configurations etc already configured?
It's possible to make such a disk with nixos, mayby other distros too.
